python/ray/ml/train/data_parallel_trainer.py

```diff
@@ -206,6 +212,9 @@ def __init__(
             resume_from_checkpoint=resume_from_checkpoint,
         )
 
+    def _validate_attributes(self):
+        super()._validate_attributes()
+
```

Comment:
Is this needed?

Reply:
Yes, otherwise the parent classes validation is not executed

Reply:
Hmm if we're only calling just the super class, then it doesn't seem necessary?

Reply:
I don't get it - we're doing more validation under the super call?

Reply:
Like in lines 218ff

Reply:
Ohh I see, sorry that didn't show up in the diff. Yes you're right, there's more validation being done below